source/educators/references/advanced_features/planning_content_reuse.rst

```diff
@@ -15,38 +14,38 @@ your development operations (DevOps) team for information about the
 website to use. At some sites, a completely separate Open edX instance, with
 a different Studio website, is set up to be the LTI tool provider.
 
-When you create links to edX course content in your external LMS, you can link
+When you create links to a course content in your external LMS, you can link
 to components individually, to all of the content in a unit, or to all of the
 content in a subsection.
 
 As you plan which parts of the course you want to reuse, note the following
 considerations.
 
-* Some edX content can be confusing to learners when it appears in the context
-  of an external LMS. For example, in some configurations, edX course
-  discussions identify learners by their internally assigned edX IDs instead of
+* Some content can be confusing to learners when it appears in the context
+  of an external LMS. For example, in some configurations, a course in an Open edX instance
```

Comment:
"a course in an Open edX instance discussions" would be better stated "Open edX course discussions"